### DISCUSSION TITLE
Change The Archived to All_POSTS

### DISCUSSION THREAD

**Discussion by 0xRyuzak1** [techno-tim/techno-tim.github.io#277] [Questions] [answered]
Hello 

Can you please help me how to change the Archived to All_POSTS in the side bar as you did

**Comment by timothystewart6** [comment, marked as answer]
Sure!  Check this out!  I created it and then included it 

https://github.com/techno-tim/techno-tim.github.io/blob/master/_tabs/all-posts.md?plain=1
